app/models/mail_chimp.rb

```diff
@@ -0,0 +1,57 @@
+require 'sqlite3'
+require 'gibbon'
+
+class MailChimp
+  API_KEY = ENV['API_KEY']
+  LIST_ID = '48af1e84e4'
+
+  def fetch_latest_entry(entry)
+    if entry.competition_id == 2
```

Comment:
That's to create a dummy name for an entry that has an email address only.  The MailChimp API seems to expect a name, so I was playing it safe.